javascript-modal/main.js

```diff
@@ -0,0 +1,15 @@
+var $openModalBtn = document.querySelector('.open-modal-btn');
+var $modalContainer = document.querySelector('.modal-container');
+var $noBtn = document.querySelector('.no-btn');
+
+function showModal(event) {
+  $modalContainer.classList.replace('hidden', 'show');
+}
+
+function hideModal(event) {
+  $modalContainer.classList.replace('show', 'hidden');
```

Comment:
Now that you don't have a `show` class anymore, you should revise this.

Reply:
doesn't `classList.replace` replace class "hidden" with class "show"?